src/argilla/server/schemas/v1/datasets.py

```diff
@@ -145,11 +145,27 @@ class TextQuestionSettings(BaseModel):
     use_markdown: bool = False
 
 
+class UniqueValuesCheckerMixin(BaseModel):
+    @root_validator
+    def check_unique_values(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        options = values.get("options", [])
+        seen = set()
+        duplicates = set()
+        for option in options:
```

Comment:
```python
options = [opt.value for opt in values.get("options", [])]
if len(options) != len(set(options)):
    raise ValueError(f"Option values must be unique, found duplicates: {duplicates}")
```

Reply:
yes, I was thinking to do it that way, but then we cannot provide the list of duplicate values in the exception message (it might be not important).

Reply:
Does not matter much. Whatever you prefer:)

Reply:
For me it is clear enough to read, but yes, it might be better to use the "long" syntax to make it clearer.